eng/common/scripts/Get-AADIdentityFromGithubUser.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.DESCRIPTION
+Get the corresponding ms alias from github identity
+.PARAMETER AadToken
+The aad access token.
+.PARAMETER GithubName
+Github identity. E.g sima-zhu
+.PARAMETER ContentType
+Content type of http requests.
+.PARAMETER AdditionalHeaders
+Additional parameters for http request headers in key-value pair format, e.g. @{ key1 = val1; key2 = val2; key3 = val3}
+#>
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$TenantId,
+  
+  [Parameter(Mandatory = $true)]
+  [string]$ClientId,
+  
+  [Parameter(Mandatory = $true)]
+  [string]$ClientSecret,
+
+  [Parameter(Mandatory = $true)]
+  [string]$GithubUser
+)
```

Comment:
Please add `Set-StrictMode -Version 3` that will help catch a number of PS errors.